### PR TITLE
Continuos integration

### PR DESCRIPTION
**Pull request by alejandro-bp**
CI setup

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          cache: 'npm'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Code style
+        run: npm run lint
+
+      - name: UI E2E Tests
+        uses: cypress-io/github-action@v2
```
Comment (holgiosalos) [inline .github/workflows/main.yml:25]: To fix the failing job you can update the version, to use v4, or add a parameter in the with section, to specify the config file:
`config-file: cypress.config.js`

Comment (alejandro-bp) [reply .github/workflows/main.yml]: I've update the version to v4 and solve the issue

File: package.json [hunk 2]
```diff
@@ -24,6 +26,13 @@
   },
   "homepage": "https://github.com/alejandro-bp/cypress-training#readme",
   "devDependencies": {
-    "cypress": "^10.3.0"
```
Comment (holgiosalos) [inline package.json:27]: make sure you have well configured the version you want to use:
- the cypress.config.js file is in the new format for cypress 10.
- the spec.js format of the tests file names is the used in version 9. but the format in version 10 is .cy.js

The workshop is documented for version 9, but if you want, you can use version 10, just that config files format is a little bit different, but searching in the doc you will find how to handle the issues. If you want to  continue using version 9, then change the cypress.config.js file to use the format of version 9.

File: cypress.config.js [hunk 3]
```diff
@@ -4,8 +4,5 @@ module.exports = defineConfig({
   e2e: {
     "video": false,
     "screenshotOnRunFailure": false,
-    setupNodeEvents(on, config) {
-      // implement node event listeners here
-    },
```
Comment (holgiosalos) [inline cypress.config.js:7]: you don't need to remove this

File: cypress.config.js [hunk 4]
```diff
@@ -4,8 +4,5 @@ module.exports = defineConfig({
   e2e: {
     "video": false,
```
Comment (holgiosalos) [inline cypress.config.js]: `video` without quotes

File: cypress.config.js [hunk 5]
```diff
@@ -4,8 +4,5 @@ module.exports = defineConfig({
...
     "screenshotOnRunFailure": false,
```
Comment (holgiosalos) [inline cypress.config.js]: `screenshotOnRunFailure` without quotes

File: cypress.config.js (hunk 3 above)
Comment (holgiosalos) [reply cypress.config.js:7]: nevermind I'm seeing you are using version 9

File: cypress.config.js (hunk 4 above)
Comment (holgiosalos) [reply cypress.config.js]: nevermind I'm seeing you are using version 9

**Review by holgiosalos — COMMENTED** [review]
cypress.config.js should be in version 9 format. 
.nvmrc should contain the version of node that you are using

**Review by manuelq12 — CHANGES_REQUESTED** [review]
Alejandro, i believe that it would be better to erase the files you aren't using.
The examples created by cypress would be a good idea

**Review by manuelq12 — APPROVED** [review]

**Comment by manuelq12** [comment]
@alejandro-bp I'm not a colaborator, can you add me please?

**Pull request merged**